query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/order_and_pagination/order_by_dependent_pagination.rs

```diff
@@ -229,9 +228,9 @@ mod order_by_dependent_pag {
     }
 
     // "[Circular with differing records] Ordering by related record field ascending" should "work"
-    // TODO: Does not work on MySQL & SQLite. Figure out why?
     // TODO(julius): should enable for SQL Server when partial indices are in the PSL
-    #[connector_test(exclude(SqlServer))]
+    // TODO(flavian): Does not work on MySQL & SQLite. Figure out why?
+    #[connector_test(exclude(SqlServer, MySql, Sqlite))]
```

Comment:
Any ideas why MySql and Sqlite have stopped working here or have they always been flaky?